### DISCUSSION TITLE
Flesh out the DataModel to new library?

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#243] [Ideas]
I found the cannonical PHP https://github.com/wmde/WikibaseDataModel today. We don't have a cannonical Python implementation yet.

I recommend fleshing out the DataModel to avoid duplicate code between this library and pywikibot (which does not support e.g. lexemes yet)

**Comment by LeMyst** [comment]
I think it's a good idea and can be easily extracted from the v0.12 who is built on this model.
I'll not do it now or in a near future because I want to release the v0.12 :)
